db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/StatementPermission.java

```diff
@@ -144,7 +144,7 @@ public String getObjectType()
         throws StandardException
 	{
 		DataDictionary dd = lcc.getDataDictionary();
-		TransactionController tc = lcc.getTransactionExecute();
+		/*TransactionController tc = */ lcc.getTransactionExecute();
```

Comment:
Does `lcc.getTransactionExecute()` has some side effects we are relying on here? If so, maybe we should leave a comment.

Reply:
you're right, we can remove this line. done